src/build/GhosttyDist.zig

```diff
@@ -36,6 +36,15 @@ pub fn init(b: *std.Build, cfg: *const Config) !GhosttyDist {
         "--format=tgz",
     });
 
+    // embed the Ghostty version in the tarball
+    {
+        const version = b.addWriteFiles().add(".version", b.fmt("{}", .{cfg.version}));
```

Comment:
I really like this idea, but just want to bikeshed on the filename. Is there any other examples of projects doing this? I'll take a look too.

Reply:
In most projects I am a part of, the filename is `VERSION`. If you search `files('VERSION')` on GitHub, you'll get a pretty fair amount of search results back.

That would find meson build files most likely.

Reply:
Changed filename to VERSION